modules/planning/lattice/trajectory_generation/trajectory1d_generator.cc

```diff
@@ -90,33 +108,19 @@ void Trajectory1dGenerator::GenerateSpeedProfileForStopping(
   auto end_conditions =
       end_condition_sampler_.SampleLonEndConditionsForStopping(stop_point);
 
-  for (const auto& end_condition : end_conditions) {
-    auto ptr_trajectory1d = std::make_shared<LatticeTrajectory1d>(
-        std::shared_ptr<Curve1d>(new QuinticPolynomialCurve1d(
-            init_lon_state_, end_condition.first, end_condition.second)));
-
-    ptr_trajectory1d->set_target_position(end_condition.first[0]);
-    ptr_trajectory1d->set_target_velocity(end_condition.first[1]);
-    ptr_trajectory1d->set_target_time(end_condition.second);
-    ptr_lon_trajectory_bundle->push_back(ptr_trajectory1d);
-  }
+  // Use the common function to generate trajectory bundles.
+  GenerateTrajectoryBundlesWithCondition(init_lon_state_, end_conditions,
+                                         ptr_lon_trajectory_bundle);
```

Comment:
Should not use the common function here, because quintic polynomial is used here

Reply:
The common function uses different initial and end conditions to generate curve bundles. So it can be used here. Please check the following line in the common function  carefully.  
```
auto ptr_trajectory1d = std::make_shared<LatticeTrajectory1d>(
        std::shared_ptr<Curve1d>(new QuarticPolynomialCurve1d(
            init_condition, end_state, end_condition.second)));
```

Reply:
OK. I'll modify it and submit again. Thank you.